app/jenkins-for-jira-ui/src/components/CopiedToClipboard/CopyToClipboardContent.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { Ref } from 'react';
+
+const HELP_LINK = 'https://app.contentful.com/spaces/zsv3d0ugroxu/entries/4ccCes4jpnMlSVtI4Eqre9?focusedField=body';
+
+type WebhookGuideContentProps = {
+	siteName: string,
+	divRef: Ref<HTMLDivElement> | undefined,
+	webhookUrl: string
+};
+
+export const WebhookGuideContent = ({ divRef, webhookUrl, siteName }: WebhookGuideContentProps) => {
+	return (
+		<div ref={divRef} data-testid="copy-webhook-guide">
+			<p>
+				Here's (almost) everything you need to set up Jenkins for Jira (official) for my Jenkins site.
+			</p>
+			<br />
+
+			<p>
+				Step-by-step guide (this will explain how to use the subsequent items):&nbsp;
+				<a href={HELP_LINK}>{HELP_LINK}</a>
+			</p>
+			<p></p>
```

Comment:
Turns out, if you add an anchor tag to a p tag, the br doesn't apply. Could only get this to work by dropping in a another p tag to sit in between.